.github/workflows/deploy-non-prod-web.yml

```
@@ -10,16 +10,28 @@ on:
         default: 'QA'
         type: choice
         options:
-          - QA
+          - USQA
           - EUQA
           - USPROD
+          - EUPROD
+      tag:
+        description: "Branch name to deploy (examples: 'main', 'feature/sm')"
+        type: string
+        default: main
 
   workflow_call:
     inputs:
       environment:
         description: 'Environment'
-        default: 'QA'
+        default: 'USQA'
+        type: string
+      tag:
+        description: "Branch name to deploy (examples: 'main', 'feature/sm')"
         type: string
+        default: main
```

```suggestion
      tag:
        description: "Branch or Tag name to deploy (examples: 'main', 'feature/sm', 'web-v2023.12.0')"
        type: string
        default: main
```
🎨  Small improvement to the `description` to help be a bit clearer to the user.

---

Would you happen to know if the actions support tags? 🤔 It seems like [download-artifacts](https://github.com/bitwarden/gh-actions/tree/main/download-artifacts) does not.